maintainers/team-list.nix

```
@@ -267,6 +485,35 @@ with lib.maintainers; {
     scope = "Maintain Red Code Labs related packages and modules.";
   };
 
+  release = {
+    members = [
+      mic92 # Release Engineer
```

Not sure where the Release engineer came from.
```suggestion
      mic92 # Release editor
```

---

From my memory probably… I removed all users in this team and switched to the GitHub team so we don't need to update this for every release (not pushed yet) ;)